app/src/main/kotlin/org/cru/godtools/ui/tools/ToolCardLayouts.kt

```diff
@@ -98,6 +99,8 @@ fun PreloadTool(tool: Tool) {
 fun LessonToolCard(
     toolCode: String,
     modifier: Modifier = Modifier,
+    isLessonDashboard: Boolean = false,
+    lessonViewModel: LessonsViewModel = viewModel(),
```

Comment:
I'm not a fan of accessing the `LessonsViewModel` from here, this goes back to the issue I brought up under the `FilterMenuItem` about cross-feature usage. Another issue that this creates is that we are now creating a `LessonsViewModel` wherever we use the `LessonToolCard` composable, which includes on the favorites UI.

Long term this will need to be updated to be usable by circuit, but a better short-term solution would to be to pass in an optional `inLanguage: Locale? = null` parameter that is prefered over the `appLanguage` in the composable.